vulture/core.py

```diff
@@ -206,15 +255,23 @@ def exclude(name):
                     self.scan(module_string, filename=path)
 
     def report(self):
-        def file_lineno(item):
+        def by_size(item):
+            return item.size
+
+        def by_name(item):
             return (item.filename.lower(), item.lineno)
         unused_item_found = False
         for item in sorted(
                 self.unused_funcs + self.unused_imports + self.unused_props +
                 self.unused_classes + self.unused_vars + self.unused_attrs,
-                key=file_lineno):
-            print("%s:%d: Unused %s '%s'" % (
-                _format_path(item.filename), item.lineno, item.typ, item))
+                key=by_size if self.sort_by_size else by_name):
+            if self.sort_by_size:
+                print("%s:%d: Unused %s '%s' size:%d" % (
+                    _format_path(item.filename), item.lineno, item.typ,
+                    item, item.size))
```

Comment:
In my opinion, `--sort-by-size` should not have an argument and the output should be sorted in ascending order by size (this is the default sorting order, see e.g. `sort -n`).